### ISSUE TITLE
Server crash 3.0.4 update

### ISSUE THREAD

**Issue by SnowMntGaming** [Darkosto/SevTech-Ages#1500]
Something keep going very wrong everytime I try to use 3.0.4 of this pack. https://pastebin.com/JjT6cu5v
Here's the log. The server just keeps looping and I can't get it to stop trying to launch again. Please help asap. My server down because of this.
* Modpack Version: 3.0.4
* Java Version: 
* Launcher Used: Twitch
* Server/LAN/Single Player: Server
* Resourcepack Enabled?: no
* Optifine Installed?: no

**Comment by SnowMntGaming** [comment]
No additional mods
I don't know the java version the server hosting uses, my options are 7 and 8. Which should I be using?
Nitrado is the hosting
I think the operating system in Linux
Forge version is 14.23.2.2635

I'm attempting reuploading all the files from the 'server pack'for 3.0.4 now, I've done this twice before and same issue. Hopefully it works now.

**Comment by SnowMntGaming** [comment]
Config - Journey maps - libraries - mods - netherex - resources - scripts - structures - minecraft_server1.12.2 - Forges (Which for my serve hosting must be renamed to) minecraft_server.jar

Those are the files I transferred in, I deleted everything and uploaded those from the server pack that is provided for 3.0.4 
I'm using java 7, World gen quark_realistic
Exact same error. I start the server it says 'server says server is starting' after a second or two it says
'server is shutting down' I can't even get log files. I have to copy from console.
